lab_1/main.py

```diff
@@ -25,7 +38,13 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
```

Comment:
There should be checks for input arguments. Smth like:
`if not isinstance(input_arg_name, expected_type):`
`do something here`